pages/settings.js

```diff
@@ -50,6 +53,10 @@ export default function SettingForm() {
           },
         });
         setLoading(false);
+        toastNoti({
+          message: "Settings saved",
+          duration: 3000,
```

Comment:
duration should be configurable.  for consistency, no need to pass it here, set it once at toastNoti function